wallet/test/Test/Pos/Util/MnemonicsSpec.hs

```diff
@@ -0,0 +1,64 @@
+module Test.Pos.Util.MnemonicsSpec (spec, Entropy(..)) where
+
+import           Universum
+
+import           Data.ByteString.Char8 (pack)
+import           Data.Set (Set)
+import           Test.Hspec (Spec, it, shouldSatisfy, xit)
+import           Test.Hspec.QuickCheck (modifyMaxSuccess, prop)
+import           Test.QuickCheck (Arbitrary (..), forAll, property)
+import           Test.QuickCheck.Gen (oneof, vectorOf)
+
+import           Pos.Util.BackupPhrase (BackupPhrase (..), safeKeysFromPhrase)
+import           Pos.Util.Mnemonics (defMnemonic, fromMnemonic, toMnemonic)
+import           Pos.Wallet.Web.ClientTypes.Functions (encToCId)
+import           Pos.Wallet.Web.ClientTypes.Types (CId)
+
+import qualified Data.Set as Set
+
+
+spec :: Spec
+spec = do
+    it "No example mnemonic" $
+        fromMnemonic defMnemonic `shouldSatisfy` isLeft
+
+    it "No empty mnemonic" $
+        (fromMnemonic "") `shouldSatisfy` isLeft
+
+    it "No empty entropy" $
+        (toMnemonic "") `shouldSatisfy` isLeft
+
+    modifyMaxSuccess (const 10000) $ prop "toMnemonic >=> fromMnemonic = Right" $
+        \(Entropy ent) -> (toMnemonic ent >>= fromMnemonic) == Right ent
+
+    -- Turn xit -> it to run, and go get a looooong coffee.
+    xit "entropyToWalletId is injective, (very long to run, used for investigation)"
```

Comment:
`pendingWith` would allow this comment to be part of the pending test output

Reply:
Except that we can't use `pendingWith` inside a QuickCheck property. It seems to be designed to go only on "standard" hspec tests ... Unless I am missing something :/

Reply:
oh, good point :(

hm, might be able to do

```haskell
it "foo" $ do
  pendingWith "asdf"
  property $ ...
```

but not a big deal really